shogun-config/src/main/resources/application-base.yml

```diff
@@ -13,6 +13,8 @@ spring:
   main:
     banner-mode: off
   jpa:
+    # Show SQL statements
+    show-sql: true
```

Comment:
intended?

Reply:
Actually it is as this will show us:

  * if hibernate creates (unwanted) complex queries
  * if a query is being issued to the database even if it is not intended

during development.